src-java/reroute-topology/reroute-storm-topology/src/main/java/org/openkilda/wfm/topology/reroute/model/FlowThrottlingData.java

```diff
@@ -32,20 +32,22 @@
     private Integer priority;
     private Instant timeCreate;
     private Set<IslEndpoint> affectedIsl;
+    private String reason;
 
     @VisibleForTesting
     public FlowThrottlingData(String correlationId, Integer priority) {
-        this(correlationId, priority, Instant.now(), Collections.emptySet());
+        this(correlationId, priority, Instant.now(), Collections.emptySet(), null);
```

Comment:
does it make sense to remove this constructor at all?

Reply:
Reworked code to remove this constructor.